infrastructure/ansible/roles/traffic-router/defaults/main.yml

```diff
@@ -177,4 +177,4 @@ tr_crontab:
   access_cleanup:
     schedule: '10 */4 * * *'
     user: root
-    job:  /bin/find /opt/traffic_router/var/log/access.log.* -mmin +$((60*48)) -exec rm -f {} \; > /dev/null 2>&1
+    job:  /bin/find {{tr_log_dir}}/access.log.* -mmin +$((60*48)) -exec rm -f {} \; > /dev/null 2>&1
```

Comment:
When you use a var like that, you have to surround it with double-quotes I believe.

Reply:
yea missed that. it's fixed now.